Index: linux-4.14.121/scripts/Makefile.modpost
===================================================================
--- linux-4.14.121.orig/scripts/Makefile.modpost
+++ linux-4.14.121/scripts/Makefile.modpost
@@ -61,8 +61,8 @@ kernelsymfile := $(objtree)/Module.symve
 modulesymfile := $(firstword $(KBUILD_EXTMOD))/Module.symvers
 
 # Step 1), find all modules listed in $(MODVERDIR)/
-MODLISTCMD := find $(MODVERDIR) -name '*.mod' | xargs -r grep -h '\.ko$$' | sort -u
+MODLISTCMD := find $(MODVERDIR) -name '*.mod' | env -i PATH="$$PATH" $$(which xargs) -r grep -h '\.ko$$' | sort -u
 __modules := $(shell $(MODLISTCMD))
 modules   := $(patsubst %.o,%.ko, $(wildcard $(__modules:.ko=.o)))
 
 # Stop after building .o files if NOFINAL is set. Makes compile tests quicker
